### PR TITLE
doc: Adding stub getting started guide

### PR DESCRIPTION
**Pull request by brianredbeard**
The docs mention little about how to get started with Seesaw. This
begins the process of documenting what it takes to get the software
running as well as the problems it solves and (some) of the ones it
doesn't.

### CODE REVIEW

File: doc/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+Fedora/RHEL/CentOS users should note that this service is only endowed by default with the
+kernel capability `CAP_SYS_MODULE` which is insufficient for loading the `ip_vs`
+kernel module.
+
+From there users should have the sufficient permissions in place to start the
+service.  For testing purposes this can be started as follows:
+
+```
+$ seesaw_watchdog -logtostderr -v=10
```
Comment (baptr) [inline doc/getting_started.md]: -v=10 is probably overkill. I don't think the watchdog even has any verbose logs statements.

File: doc/getting_started.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+switching within the Linux kernel.
+  * `nf_conntrack_ipv4` - Used by the Linux kernel to maintain the state of all
+logical network connections and relate packets to the sessions.
+  * `dummy` - A virtual network device which can have routes assigned but will
+transfer no packets.
+
+Each of these kernel modules should be loaded before the Seesaw process is
+started.  Most users will find this easiest to achieve through the use of the
+`/etc/modules.load.d` mechanism.  To load each of these automatically with the
+built in `systemd` mechanisms run the following commands:
```
Comment (baptr) [inline doc/getting_started.md]: seems a bit odd that we're shipping an upstart service config, but documenting systemd module loading. Maybe s/Most users/systemd users/ ?

Comment (brianredbeard) [inline doc/getting_started.md]: Quite possibly.  I hadn't even gotten to the point of worrying about files for the init system yet.  Once I get this a bit further along I'm happy to contribute systemd service unit files.  As for the usage of `/etc/modules.load.d` that should be valid for most distros with the notable exception of Gentoo w/o systemd and Slackware.  Even where Ubuntu calls `rc.local` it's currently done via systemd IIRC.  Regardless, not trying to start a religious war. I'll throw in some clarifying comments about manually running `modprobe` too.

**Comment by baptr** [comment]
Thanks!

**Pull request merged**